docs/src/content/docs/de/reference/frontmatter.md

```diff
@@ -18,30 +19,37 @@ Willkommen auf der Info-Seite!
 
 ### `title` (erforderlich)
 
-**type:** `string`
+**Typ:** `string`
 
 Du musst für jede Seite einen Titel angeben. Dieser wird oben auf der Seite, in Browser-Tabs und in den Seiten-Metadaten angezeigt.
 
 ### `description`
 
-**type:** `string`
+**Typ:** `string`
 
 Die Seitenbeschreibung wird für die Metadaten der Seite verwendet und wird von Suchmaschinen und in der Vorschau von sozialen Medien angezeigt.
 
+### `slug`
+
+**Typ**: `string`
+
+Setzt den Slug der Seite außer Kraft. Siehe [„Benutzerdefinierte Slugs definieren“](https://docs.astro.build/de/guides/content-collections/#benutzerdefinierte-slugs-definieren) in der Astro-Dokumentation für weitere Details.
```

Comment:
```suggestion
Setzt den Slug der Seite außer Kraft. Siehe [„Benutzerdefinierte Slugs definieren“](https://docs.astro.build/de/guides/content-collections/#defining-custom-slugs) in der Astro-Dokumentation für weitere Details.
```

This page is not translated in the Astro Docs so we might as well link to the correct anchor until then.

Reply:
Oh, okay, I thought I can just link to the future... I did the same mistake on many other commits then...

Reply:
Do you (@HiDeoo ) think its a good idea if I just change this common error in all my PRs and make a new issue on the Starlight repo with all the links that needs to be fixed in the future, cause I surely cannot remember them all...?

Reply:
I think it's better to actually fix them in PRs if they're spotted like this one and if later some broken anchors are found that may have been missed, then we can fix them at that time.